### PR TITLE
[BBC-9088] Storybook: TheVoice.MDX

### PR DESCRIPTION
**Pull request by Tip-Sy**
## Description

Migrate TheVoice to new storybook

## How it was tested

Run locally (see screenshot)

Capture d’écran 2020-07-21 à 15 51 21

### CODE REVIEW

File: src/theVoice/TheVoice.style.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  ${normalizeHorizontally};
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
+    text-align: center;
+    font-size: var(--font-size-desktop);
+  }
+`
+
+export default StyledTheVoice
```
Comment (gabrielagabriel) [inline src/theVoice/TheVoice.style.tsx:27]: no need to default export, better keep it as a named export

Comment (Tip-Sy) [reply src/theVoice/TheVoice.style.tsx]: You're right, done! :)

File: src/theVoice/TheVoice.story.mdx [hunk 2]
```diff
@@ -1,26 +1,41 @@
-## The Voice
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
 
-<!-- STORY -->
+import { TheVoice } from './index'
+import { color } from '../_utils/branding'
+
+<Meta title="TheVoice" />
```
Comment (gabrielagabriel) [inline src/theVoice/TheVoice.story.mdx:6]: Add a category
```suggestion
<Meta title="Text|TheVoice" />
```

File: src/theVoice/TheVoice.style.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  /* 40px is an exception on font scale  */
+  --font-size-desktop: 40px;
+
+  margin: 0;
+  padding: ${space.xl} 0 var(--space-bottom);
+  white-space: pre-line;
+
+  ${normalizeHorizontally};
+
+  &.kirk-thevoice--inverted {
```
Comment (mcellier) [inline src/theVoice/TheVoice.style.tsx:20]: Note: i used a class instead of the `isInverted` prop as `Title` doesn't handle this prop (TS was complaining)

Comment (gabrielagabriel) [reply src/theVoice/TheVoice.style.tsx:20]: Some day refactor then.. should we create a ticket for that?

**Review by gabrielagabriel — APPROVED** [review]

File: src/theVoice/TheVoice.style.tsx (hunk 3 above)
Comment (mcellier) [reply src/theVoice/TheVoice.style.tsx:20]: We have these kind of things in lots of components. We'll see later as we already have more importants concerns IMHO

Comment (mcellier) [reply src/theVoice/TheVoice.style.tsx:20]: Plus I don't think we want this prop in `Title`

**Review by wakooka — APPROVED** [review]

**Pull request merged**